boilerplate/.maestro/Login.yaml

```diff
@@ -0,0 +1,18 @@
+#flow: Login
+#intent:
+# Open up our app and use the default credentials to login
+# and navigate to the demo screen
+
+appId: com.helloworld # the app id of the app we want to test
+# You can find the appId of an Ignite app in the `app.json` file
+# as the "package" under the "android" section and "bundleIdentifier" under the "ios" section
+---
+- clearState # clears the state of our app (navigation and authentication)
+- launchApp # launches the app
+- assertVisible: "Sign In"
+- tapOn:
+    text: "Tap to sign in!"
+- assertVisible: "Your app, almost ready for launch!"
+- tapOn:
+    text: "Let's go!"
+- assertVisible: "Components to jump start your project!"
```

Comment:
```suggestion
- assertVisible: "Components to jump start your project!"

// @demo remove-file

```

Reply:
Threw up a commit to fix this since I was already messing around with it locally